includes/RestApi/Skip404Controller.php

```diff
@@ -79,21 +79,18 @@ public function set_options( $request ) {
 
 			switch ( $field['id'] ) {
 				case 'skip404':
-					$result = update_option( Skip404::OPTION_NAME, $field['value'] );
+					$bool_value = filter_var( $field['value'], FILTER_VALIDATE_BOOLEAN );
+					update_option( Skip404::OPTION_NAME, $bool_value );
 					break;
 
 				default:
-					break;
-			}
-
-			if ( false === $result ) {
```

Comment:
@abhijitb Seems like we’re returning success even if the option didn’t actually save, is that okay? Just wondering since it might mess up the `.htaccess` rules if the value isn’t properly synced.